README.md

```diff
@@ -311,7 +311,7 @@ sudo easy_install pip
 sudo pip install -r python_requirements.txt
 ```
 
-If previous command fails because of gdal, comment the dependency: `# gdal==1.10.0`.
+If previous command fails because of gdal, comment the dependency: `# gdal==1.10.0`. Try `sudo easy_install GDAL` instead (GDAL is needed for SHP normalization, for example).
```

Comment:
Yup, I must admit that I did a first check without much hope and didn't specified more later. Can't `python_requirements.txt` be just updated?